Modules/_ctypes/_ctypes.c

```diff
@@ -794,6 +794,10 @@ CDataType_from_buffer(PyObject *type, PyObject *args)
 PyDoc_STRVAR(from_buffer_copy_doc,
 "C.from_buffer_copy(object, offset=0) -> C instance\ncreate a C instance from a readable buffer");
 
+static inline PyObject *
+_GenericPyCData_new(ctypes_state *st,
```

Comment:
Could you use e.g. `GenericPyCData_new_st`?

Identifiers with a leading underscore followed by a capital letter are reserved according to the C standard.
We do use `_Py` all over (which will probably bite us in the future). But let's not add `_G`.

Reply:
I've changed to snake_cases as an example.